jenkins-build-pack-api/Jenkinsfile

```diff
@@ -92,9 +92,11 @@ node() {
 
 
 	def configPackBBURL = "http://" + var_bitbucket_base + "/scm/" + var_repo_core + "/serverless-config-pack.git"
+	def jazzBuildModuleURL = "http://${var_bitbucket_base}/scm/"+ var_repo_core +"/jazz-build-module.git"
 
 	def config
-
+	def configObj
```

Comment:
Similar comments as in delete-serverless-service-build-pack/Jenkinsfile

Reply:
In cooperated review comments